client/public/account/update_recipe.html

```diff
@@ -437,10 +437,15 @@ <h2>Update Recipe</h2>
         </div>
 
         <div class="form-group">
-          <label for="userRecipeImage">Recipe Image</label>
+          <label for="userRecipeImage">Recipe Image - Please select a new image or reupload the original.</label>
           <input type="file" id="userRecipeImage" name="userRecipeImage" class="update-recipe-input" accept="image/*" />
         </div>
 
+        <div class="form-group">
```

Comment:
Could you please add a screenshot of what the form looks like?

Reply:
![image](https://github.com/user-attachments/assets/9e08dc53-e5ef-436a-b6ee-ce39510804a2)